explorer-backend/fly/migration/migration.go

```diff
@@ -164,5 +170,44 @@ func Run(db *mongo.Database) error {
 		return err
 	}
 
+	statsIndexes := []mongo.IndexModel{
+		{Keys: bson.D{{Key: "date", Value: 1}}},
+		{
+			Keys: bson.D{
+				{Key: "date", Value: 1},
+				{Key: "emitterChain", Value: 1},
+			},
+		},
+		{
+			Keys: bson.D{
+				{Key: "date", Value: 1},
+				{Key: "emitterChain", Value: 1},
+				{Key: "emitterAddr", Value: 1},
+			},
+		},
+		{
```

Comment:
Same as a previous comment, seems that we only need to create the first and the last compound indexes